Chirp/ViewController.swift

```diff
@@ -7,13 +7,60 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+class ViewController: UITabBarController {
```

Comment:
Could we please create a new file and class instead? You can call it `HomeTabBarController`